breaking/entrypoint.sh

```diff
@@ -31,12 +31,12 @@ readonly exclude_elements="$8"
 readonly composed="$9"
 readonly output_to_file="${10}"
 
-echo "running oasdiff breaking... base: $base, revision: $revision, fail_on_diff: $fail_on_diff, include_checks: $include_checks, include_path_params: $include_path_params, deprecation_days_beta: $deprecation_days_beta, deprecation_days_stable: $deprecation_days_stable, exclude_elements: $exclude_elements, composed: $composed, output_to_file: $output_to_file"
+echo "running oasdiff breaking... base: $base, revision: $revision, fail_on: $fail_on, include_checks: $include_checks, include_path_params: $include_path_params, deprecation_days_beta: $deprecation_days_beta, deprecation_days_stable: $deprecation_days_stable, exclude_elements: $exclude_elements"
```

Comment:
why did you delete `composed: $composed, output_to_file: $output_to_file"`?

Reply:
@jgill-shipwell could you please check (this PR)[https://github.com/oasdiff/oasdiff-action/pull/52] and let me know if it resolves your issue?